packages/engine/src/avatar/functions/avatarFunctions.ts

```diff
@@ -170,6 +171,8 @@ export const setupAvatarProportions = (entity: Entity, vrm: VRM) => {
  * successfully loaded.
  */
 export const setupAvatarForUser = (entity: Entity, model: VRM) => {
+  if (entity == Engine.instance.localClientEntity) setComponent(entity, TransparencyDitheringComponent)
```

Comment:
i think we should add this to all avatars, would help with other being trying to block your vision in VR